data/remnant/remnant missions.txt

```diff
@@ -4634,12 +4628,412 @@ mission "Remnant: Expanded Horizons Storms 1"
+
+	npc kill
+		government "Test Dummy"
+		system "Caeculus"
+		ship "_CME Timer Ship" "Timer"
+		personality unconstrained heroic nemesis
+
+		dialog `The star flares bright against your cockpit windows, which automatically darken to protect your fragile eyes from its deadly rays. At first, the light seems to dim, but within moments you watch a wave of incandescent plasma paint the space before you. Intellectually, you know that you are safe behind your ship's shields and armor, but the grand scale and majesty of the ejection puts all else to shame. Several minutes pass in silence as you both watch the swirling currents of plasma outside. You and Dawn nearly jump out of your seats as the laboratory equipment loudly chimes, signalling that it has collected the required data. You can now go on to Ildaria to observe a coronal mass ejection there.`
+
+	on enter Ildaria
+		conversation
+			`	Again, Dawn sets up the laboratory equipment before taking her seat. "We can learn a significant amount by simply measuring the frequency of the ejection events in each system. Remote observations show that they are less frequent here, but actually being here should give us much more accurate measurements."`
+
+			`	"Remember," she says, "straying too close to the star when the ejection event starts could be dangerous."`
+
+	npc kill
+		government "Test Dummy"
+		system "Ildaria"
+		ship "_CME Timer Ship" "Timer"
+		personality unconstrained heroic nemesis
+
+		dialog `The star flares in what is now a familiar pattern, whipping long trails of crackling plasma out into the cosmos. Is it just your imagination, or is this a less violent and energetic ejection than the one in the Ember Waste? You ponder on in silence before the laboratory equipment chimes, signalling that it you can now return to <destination> with the data.`
+
+	on complete
+		payment 1000000
+		conversation
+			`	The journey back is uneventful, though you have gained some newfound appreciation for the relative serenity of most star systems. "So, how long will it take before you'll have answers?" you ask. "We can't be sure," says Dawn. "After all, we've been observing these stars for centuries. It is hoped that data outside the Ember Waste will give us a more useful control point, but even if we do identify something wrong with the local stars, it could be years or even decades before we know what exactly changed them."`
+			choice
+				`	"Do you think it could have something to do with the ion storms?"`
+					goto storms
+				`	"Could a similar 'change' have created the unstable wormholes?"`
+					goto wormholes
+
+			label storms
+			`	"It is certainly possible. There is a large flux of energy being poured into the surrounding nebulae, and the source, whatever it may be, has been active for eons. Who knows what kind of effect it may have originally had on these stars, millions of years ago?"`
```

Comment:
```suggestion
			`	"It is certainly possible. There is a large flux of energy being poured into the surrounding nebulae, and the source, whatever it may be, has been active for eons. Who knows what kind of effect it may have originally had on these stars, millions of years ago?`
```